tasks/get_record.json

```diff
@@ -25,6 +25,10 @@
         "instance": {
             "description": "ServiceNow instance name. If your instance uri is https://dev84270.service-now.com your instance name would be dev84270.",
             "type": "Optional[String[1]]"
+        },
+        "access_token": {
```

Comment:
Hmm, I think I prefer `oauth_token` instead of `access_token` b/c it's more explicit. I also don't know if people usually think of `oauth_token` as `access_token` -- if they do, then maybe `access_token` is OK.

Reply:
Servicenow calls them access tokens, but they also seems to have a lot of non-standard oauth flows.

Reply:
Ah ok. If ServiceNow calls them that then I am 👍